cgmanifest.json

```diff
@@ -3699,8 +3699,8 @@
         "type": "other",
         "other": {
           "name": "fontawesome-fonts",
-          "version": "4.7.0",
-          "downloadUrl": "https://github.com/FortAwesome/Font-Awesome/archive/refs/tags/v4.7.0.tar.gz"
+          "version": "6.5.1",
+          "downloadUrl": "https://github.com/fortawesome/font-awesome/archive/refs/tags/6.5.1.tar.gz"
```

Comment:
I think the change in case of the url might have caused the `CGManifest` check to fail.

Reply:
It appears that the Auto-upgrade pipeline is case-insensitive w.r.t. URLs leading to these kinds of issues. Thanks for pointing out. Fixed now.